fastai/optimizer.py

```diff
@@ -27,15 +39,27 @@ def freeze(self):
         assert(len(self.param_lists)>1)
         self.freeze_to(-1)
 
-    def set_freeze(self, n, rg, ignore_force_train=False):
+    def set_freeze(self,
+        n:int,
+        rg:bool, # Gradient flag for layer `n`
+        ignore_force_train=False  # Overwrites "force_train" or batch norm always trains even if frozen
+    ):
         for p in self.param_lists[n]: p.requires_grad_(rg or (state.get('force_train', False) and not ignore_force_train))
 
-    def unfreeze(self): self.freeze_to(0)
-    def set_hypers(self, **kwargs): L(kwargs.items()).starmap(self.set_hyper)
-    def _set_hyper(self, k, v):
+    @delegates(_BaseOptimizer.set_hyper)
+    def set_hypers(self,
```

Comment:
Missing docstring.

Reply:
Tried to document adding the `@delegates(_BaseOptimizer.set_hyper)` but it causes above error as on above comment https://github.com/muellerzr/fastai-docment-sprint/pull/68#issuecomment-1046355465

If cant document it with delegates, will leave without documentation.